### DISCUSSION TITLE
[Enhancement] What about Blend for Visual Studio Support MAUI

### DISCUSSION THREAD

**Discussion by jingliancui** [dotnet/maui#153] [General]
Is it possible using Blend to support MAUI's UI design (include animation etc.) as it support for WPF ?

Then it can promote the efficiency to the MVVM pattern (or the MVU).

**Comment by jeremynation-ceros** [comment]
I'd also like some clarification regarding Blend. If MAUI is replacing "all the things", including windows apps, does that mean Blend is dead?

**Reply by jingliancui** [reply]
I think maybe it can add "extension" feature like visual studio xamarin component do for Blend. It doesn't affect others.

**Reply by jingliancui** [reply]
I think it's still have lots of developer and enterprise use native app solution for their products. So I don't think Blend is dead.

**Comment by jeremynation-ceros** [comment]
I had the opportunity to ask @maddyleger1 and she gave a great answer to this question in the last WinUI Community Call: [https://youtu.be/SyLXctia1B0?t=2636](https://youtu.be/SyLXctia1B0?t=2636). Many thanks to her!

**Reply by ManuelKugelmann** [reply]
And what was the answer?

**Reply by ThaDaVos** [reply]
@jeremynation-ceros  - what was the answer? Can you tell us instead of redirecting to a video?

**Reply by UkeHa** [reply]
"Drag and Drop tools are very hard for different device sizes" and you should use platform specific sizes and blend isn't a good fit for mobile but great for desktop. So i wouldn't hold my breath.

**Comment by Schinwinkwinsky** [comment]
The winter is comming and the low-code/no-code is something that can't be ignored. We have good tools as FlutterFlow that is an amazing solution to design an app. I'm not sure, but to say that Blend "isn't a good fit for mobile" is questionable. Blend is a powerful tool and I think we can give it a chance with Maui.

**Reply by ThomasRocha82** [reply]
It's curious in a way because Silverlight had Blend support.. It worked for Web, Desktop, Tablet, and Windows Mobile. Blend supported it's XAML version and even had a built-in emulator for multiple Windows Phone models. 

I've never really bought the reasoning that it's not a good fit for those reasons. I know many that moved away from dotnet for mobile and other platforms outside of WPF because of these decisions. MAUI looks cool, sure - but a good chunk of people would rather have something more fluid to work with other than "live update". Blend has been a staple for UI design since 2008, why throw the baby out with the bathwater?

No offense meant to the team, just when comparing other options - one person I know moved his team to Embarcadero's FireMonkey and went back to Delphi so their designers could be more productive.
